providers/tests/celery/cli/test_celery_command.py

```diff
@@ -270,96 +270,196 @@ def test_run_command(self, mock_celery_app):
             ]
         )
 
+    @mock.patch("airflow.cli.commands.daemon_utils.TimeoutPIDLockFile")
+    @mock.patch("airflow.cli.commands.daemon_utils.setup_locations")
+    @mock.patch("airflow.cli.commands.daemon_utils.daemon")
+    @mock.patch("airflow.providers.celery.executors.celery_executor.app")
+    def test_run_command_daemon_v_3_below(
+        self, mock_celery_app, mock_daemon, mock_setup_locations, mock_pid_file
+    ):
+        if not AIRFLOW_V_3_0_PLUS:
```

Comment:
Don't put the whole body of the test in a conditional - use [pytest skipif](https://docs.pytest.org/en/stable/how-to/skipping.html#id1) instead.

Reply:
I used the `pytest skipif`, that was a nice tip! I am monitoring the CI, thanks!